samza-core/src/test/scala/org/apache/samza/metrics/reporter/TestJmxReporter.scala

```diff
@@ -63,7 +63,8 @@ object TestJmxReporter {
 class TestJmxReporter {
   import TestJmxReporter.url
 
-  @Test
+  // TODO: Fix in SAMZA-1194
```

Comment:
Is the fix in SAMZA-1194 or you are commenting our as a part of SAMZA-1194 (as in your title)? Please correct this.
Also, as discussed before, please assign an owner for fixing this test.